src/rulesToUnnamedParametersDefaults.ts

```diff
@@ -90,10 +91,10 @@ export class rulesNamedParametersAndDefaults {
   static parameter(
     patternId: string,
     parameter: string
-  ): ParameterSpec | undefined {
+  ): [string, any] | undefined {
```

Comment:
🤮  does it need to be any? oh well 😅

Reply:
Yes because it can be anything that can be a json ( boolean, string, object, number ).